modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -323,6 +323,63 @@ definition {
 			locator1 = "CreateObject#SELECT_OBJECT_FIELD_IN_TABLE");
 	}
 
+	@description = "Verify that it's possible to view the custom object after restarting portal"
+	@priority = "5"
+	test CanViewObjectAfterRestart {
+		property portal.acceptance = "true";
+
+		task ("Create Custom Object and Custom Field (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 163066",
+				objectName = "CustomObject163066",
+				pluralLabelName = "Custom Objects 163066");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject163066");
+		}
+
+		task ("Go to Custom Object, change the panel category key and publish") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 163066");
+
+			CreateObject.selectPanelCategoryKey(panelCategoryKey = "Control Panel > Users");
+
+			ObjectAdmin.publishObjectViaUI();
+		}
+
+		task ("Create an Entry (via API)") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject163066",
+				value = "Entry Test");
+		}
+
+		task ("Restart portal") {
+			Portlet.shutdownServer();
+
+			Portlet.startServer(deleteLiferayHome = "false");
+		}
+
+		task ("Go to Custom Object entries") {
+			Navigator.openURL();
+
+			ApplicationsMenu.gotoPortlet(
```

Comment:
Hi @andreefarias you're in the correct way but you forget to assert that the custom object is present on the panel under Control Panel > User like says step 6:  `Open the global menu and **assert the custom object is present on the panel under Control Panel** > User`